PCGen-Formula/code/src/java/pcgen/base/formula/parse/Operator.java

```diff
@@ -52,9 +52,9 @@ public String getSymbol()
 		return symbol;
 	}
 
-	private String symbol;
+	private final String symbol;
 
-	private Operator(String op)
+	Operator(String op)
```

Comment:
I don't think removing this adds value.  It may be the default, but that doesn't mean it isn't valuable for those not familiar with the default.